_src/static/sass/components/_buttons.scss

```diff
@@ -0,0 +1,389 @@
+// ------------------------------
+// edX Pattern Library: Components - Buttons
+
+// About: Contains base styling for demonstration and utility for buttons.
+// ----------------------------
+
+// #LAYOUT
+// #GLOBALS
+// #BASE (size)
+// #LARGE (size)
+// #SMALL (size)
+// #DEFAULT (style)
+// #DEFAULT-ALT (style)
+// #PRIMARY (style)
+// #SECONDARY (style)
+// #SECONDARY-ALT (style)
+// #LINK (style)
+// #ELEVATED (type)
+// #COMBO (type)
+// #ICONS (type)
+// #OVERLAY (type)
+// ----------------------------
+
+// ----------------------------
+// #GLOBALS
+// ----------------------------
+.btn {
+    @include transition(
+        color timing(xx-fast) ease-in-out 0s,
+        border-color timing(xx-fast) ease-in-out 0s,
+        background timing(xx-fast) ease-in-out 0s,
+        box-shadow timing(xx-fast) ease-in-out 0s
+    );
+
+    // Display: inline, side-by-side
+    display: inline-block;
+    border-style: $buttons-border-style;
+    border-radius: $buttons-border-radius;
+    border-width: $buttons-border-size;
+
+    // Display: block, one button per line, full width
+    &.block {
+        display: block;
+        width: 100%;
+    }
+}
+
+// ----------------------------
+// #BASE
+// ----------------------------
+.btn-base {
+    font-size: font-size(mid-large);
+    padding: rem(($baseline/1.5)) rem(($baseline*1.25));
```

Comment:
Should we be using ``spacing-vertical()`` and `spacing-horizontal()`` values instead of baseline here? I think there are merits to us staying with one way to reference our set scales of vertical/horizontal spacing.